examples/quickstart/Readme.md

```diff
@@ -14,5 +12,13 @@ export RPC_URL=<your-eth-rpc-url>
 cargo run --release --example quickstart
 ```
 
+By default, the example will trade 1 WETH -> USDC. If you want a different trade you can do:
+
+```bash
+cargo run --release --example quickstart -- --sell-token "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48" --buy-token "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599" --sell-amount 10000
```

Comment:
Ah quick question, just noticed this now, what is with this `--` after quickstart? Haven't tried to run this yet

Reply:
it's necessary for clap to recognise whatever comes next as the CLI args